libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -704,6 +704,14 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("GND_EFF_DZ", 7, NavEKF3, _baroGndEffectDeadZone, 4.0f),
 
+    // @Param: PRIMARY
+    // @DisplayName: Primary core number
+    // @Description: The code number (index in IMU mask) that will be used as the primary EKF core on startup. While disarmed the EKF will force the use of this core. A value of 0 corresponds to the first IMU in EK3_IMU_MASK.
```

Comment:
one last thing, the description says, "code" (with a "d") instead of core.

Reply:
oops, fixed, thanks